domain_alias/src/Controller/DomainAliasController.php

```diff
@@ -34,4 +35,21 @@ public function addAlias(DomainInterface $domain) {
     return $this->entityFormBuilder()->getForm($alias);
   }
 
+  /**
+   * Provides the listing page for aliases.
+   *
+   * @param \Drupal\domain\DomainInterface $domain
+   *   An domain record entity.
+   *
+   * @return array
+   *   A render array as expected by drupal_render().
+   */
+  public function listing(DomainInterface $domain) {
+    $type = \Drupal::entityTypeManager()->getDefinition('domain_alias');
+    $storage = \Drupal::entityTypeManager()->getStorage('domain_alias');
+    $list = new DomainAliasListBuilder($type, $storage);
```

Comment:
We may be able to just call the service directly. e.g.

```
$list = $this->entityManager()->getListBuilder($entity_type);
```
See EntityListController::listing().